frontend/public/components/secrets/subforms.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
```

Comment:
These components are going to be so tightly coupled to create secret, I would keep them in the same file. Then you don't need to export them since they're internal to the implementation.